website/content/docs/agent-and-proxy/agent/apiproxy.mdx

```diff
@@ -6,14 +6,24 @@ description: >-
   for Vault's API.
 ---
 
-# Vault agent API proxy
+# Vault Agent API proxy
```

Comment:
Calling it Agent proxy is a little confusing -- should we separate this out and stick with calling it "Vault Proxy" throughout? Or, if we're referring to the pre-deprecated functionality, we have called this "Vault Agent in Proxy mode"

Reply:
This is talking about the API proxy functionality present in Vault Agent. While we encourage people to use Vault Proxy's API proxy functionality, Vault Agent still has API proxy functionality in it today

Reply:
Makes sense!